src/main/java/com/microsoft/jenkins/containeragents/aci/dns/AciDnsConfig.java

```diff
@@ -0,0 +1,43 @@
+package com.microsoft.jenkins.containeragents.aci.dns;
+
+import hudson.Extension;
+import hudson.model.AbstractDescribableImpl;
+import hudson.model.Descriptor;
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.kohsuke.stapler.DataBoundSetter;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class AciDnsConfig extends AbstractDescribableImpl<AciDnsConfig> {
+
+    private List<AciDnsServer> dnsServerNames = new ArrayList<>();
```

Comment:
applies throughout file, looking at the jcasc config dns is in front of everything I don't think we need to keep nesting it?
```suggestion
    private List<AciDnsServer> serverNames = new ArrayList<>();
```

alternatively

```suggestion
    private List<AciDnsServer> nameServers = new ArrayList<>();
```

Reply:
I use `dnsServer`. That it is independent by the internals of AciDnsServer